src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -6,6 +6,16 @@
  void npe(String a, String b) {
   if (a == null) {
    System.out.println();
+   System.out.println();
+  } else {
+
+  }
+  a.length();
+ }
+
+ void npe2(String a, String b) {
+  if (a == null) {
+   System.out.println();
   } else {
```

Comment:
Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck
Severity: INFO
File: /home/bjerre/sandbox/jenkinsnode/workspace/test/violations-test/src/main/java/se/bjurr/violations/lib/example/MyClass.java L19
 

Must have at least one statement.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>